feature/bgp/static-route_bgp_redistribution/README.md

```diff
@@ -0,0 +1,653 @@
+    *   /routing-policy/defined-sets/tag-sets/tag-set/state/name
+*   Verify tag-set ```tag-set-v6``` with a tag value of ```100``` is configured
+    *   /routing-policy/defined-sets/tag-sets/tag-set/state/tag-value
+*   Verify for routing-policy ```route-policy-v6``` statement ```statement-v6``` match-set-tag is set to ```tag-set-v6```
+    *   /routing-policy/policy-definitions/policy-definition/statements/statement/conditions/match-tag-set/state/tag-set
+*   Verify for routing-policy ```route-policy-v6``` statement ```statement-v6``` match-set-options is set to ```ANY```
+    *   /routing-policy/policy-definitions/policy-definition/statements/statement/conditions/match-tag-set/state/match-set-options
+##### Validate test results
+*   Verify that the ATE does not receives the redistributed static route ```ipv6-route```
+    *   /network-instances/network-instance/protocols/protocol/bgp/rib/afi-safis/afi-safi/ipv6-unicast/loc-rib/routes/route/prefix
+
+### RT-1.27.21 [TODO: https://github.com/openconfig/featureprofiles/issues/2568]
+#### Redistribution of IPv6 static routes to BGP that matches a conditional tag should happen
+---
+##### Configure a tag-set with correct tag value to validate that the route is redistributed
+*   Configure tag-set ```tag-set-v6``` with a tag value of ```60```
+    *   /routing-policy/defined-sets/tag-sets/tag-set/config/tag-value
+    *   here we are setting correct tag value of 60, as defined in initial setup of this test, to validate that the route is now redistributed
+##### Verification
+*   Verify tag-set ```tag-set-v6``` with a tag value of ```60``` is configured
+    *   /routing-policy/defined-sets/tag-sets/tag-set/state/tag-value
+##### Validate test results
+*   Verify that the ATE receives the redistributed static route ```ipv6-route```
+    *   /network-instances/network-instance/protocols/protocol/bgp/rib/afi-safis/afi-safi/ipv6-unicast/loc-rib/routes/route/prefix
+*   Initiate traffic from ATE port-1 to the DUT and destined to ```ipv6-network``` i.e. ```2024:db8:128:128::/64```
+*   Validate that the traffic is received on ATE port-2
+
+### RT-1.27.22 [TODO: https://github.com/openconfig/featureprofiles/issues/2568]
+#### Redistribute a NULL IPv6 static routes to BGP with a next-hop configured through route-policy
+---
+##### Configure a NULL static route
+*   Configure an IPv6 static route ```ipv6-drop-route``` on DUT destined to ```ipv6-drop-network``` i.e. ```2024:db8:64:64::/64``` with the next hop set to ```DROP```
+    *   /network-instances/network-instance/protocols/protocol/static-routes/static/config/prefix
+    *   /network-instances/network-instance/protocols/protocol/static-routes/static/next-hops/next-hop/config/next-hop
+##### Configure a tag on the static route
+*   Set a tag on the ```ipv6-drop-route``` to 60
+        *   /network-instances/network-instance/protocols/protocol/static-routes/static/config/set-tag
+##### Configure BGP actions to set a next-hop
```

Comment:
In DCGate usecase, we do not need to set NH for NULL static routes as thye are going to be advertised over eBGP enly, henc NH would be overwritten anyway.
But it do not harm either.